content/doc/book/system-administration/reverse-proxy-configuration-with-jenkins/reverse-proxy-configuration-apache.adoc

```diff
@@ -275,7 +275,7 @@ A typical mod_rewrite configuration would look like this:
 # NE makes sure slashes are not re-encoded.
 # Apache does not re-encode spaces though, we ask Apache to encode it again with the B flag
 # BNP tells apache to use %20 instead of + to re-encode the space
-RewriteRule       ^/jenkins(.*)$  http://localhost:8081/jenkins$1 [P,L,NE,B=\ \,BNP]
+RewriteRule       ^/jenkins(.*)$  http://localhost:8081/jenkins$1 [P,L,NE,B=\,BNP]
```

Comment:
The RewriteRule Flags page says:

> The [B] flag instructs [RewriteRule](https://httpd.apache.org/docs/2.4/mod/mod_rewrite.html#rewriterule) to escape non-alphanumeric characters before applying the transformation.

I think that means the sequence `B=\ \,` would be interpreted as "escape the space character and the comma character".  I think the desire is to escape the space character and not escape the comma character.  However, I do not have an Apache 2.4 reverse proxy configured.

Does your proposed change work or is a space character needed immediately after the `\` character?
```suggestion
RewriteRule       ^/jenkins(.*)$  http://localhost:8081/jenkins$1 [P,L,NE,B=\ ,BNP]
```

Reply:
I have verified this change on Apache 2.4 that it did not work previously without my PR change.

However I'm not running it inside a path but rather at the root of a domain, so I'm not exactly sure if that matters or not.

Reply:
Thanks.  If that syntax works for you, then that is the best we can offer.  Thanks for the pull request!